test/Preprocessing/lit.local.cfg.py

```diff
@@ -0,0 +1,16 @@
+# -*- Python -*-
+
+import os
+import platform
+import sys
+
+import lit.formats
+from lit.llvm import llvm_config
+
+# Configuration file for the 'lit' test runner.
+
+# testFormat: The test format to use to interpret tests.
+config.test_format = lit.formats.ShTest(execute_external=False)
+
+# suffixes: A list of file extensions to treat as test files.
```

Comment:
I think you should leave a comment here explaining why you have added this file with this line. I would reference your new Issue that you created for eventually enabling these tests in this comment.